lib/modules/dosomething/dosomething_reportback/dosomething_reportback.drush.inc

```diff
@@ -1,8 +1,92 @@
 <?php
 
 /**
+ *  Implements hook_drush_command
+ */
+function dosomething_reportback_drush_command() {
+  return [
+    'ds-reportback-run-nid-fix' => [
+      'description' => 'Gives every reportback a run nid',
+      'examples' => [
+        'drush ds-reportback-run-nid-fix'
+          => 'Gives every reportback a run nid',
+      ],
+    ],
+  ];
+}
+
+/**
  * Implements hook_drush_cache_clear().
  */
 function dosomething_reportback_drush_cache_clear(&$types) {
   $types['dosomething reportback'] = 'dosomething_reportback_cache_clear_all';
 }
+
+/**
+ * Callback for ds-reportback-run-nid-fix command.
+ */
+function drush_dosomething_reportback_ds_reportback_run_nid_fix() {
+  // First retrieve every campaign NID
+  $nids = db_query("SELECT n.nid as nid
+                    FROM node n
+                    WHERE n.type = 'campaign'");
+
+  // Loop through every NID
+  foreach ($nids as $nid) {
+
+    // Get the runs for associated for this campaign & the timing
+    $runs = db_query("SELECT DISTINCT c.entity_id as run_nid, d.field_run_date_value as start_date, d.field_run_date_value2 as end_date
+                      FROM field_data_field_campaigns c
+                      INNER JOIN field_data_field_run_date d ON d.entity_id = c.entity_id AND d.language = 'en'
+                      WHERE c.bundle = 'campaign_run'
+                      AND c.language = 'en'
+                      AND c.field_campaigns_target_id =" . $nid->nid ."
+                      ORDER BY d.field_run_date_value2 ASC");
```

Comment:
Yeah on line 40/41 its single quotes, but the query is surrounded by double. i'm missing something?